maven/build_test.go

```diff
@@ -196,6 +223,35 @@ func testBuild(t *testing.T, context spec.G, it spec.S) {
 		Expect(result.BOM.Entries[0].Launch).To(BeFalse())
 	})
 
+	it("contributes mvnd distribution for API <=0.6", func() {
+		os.Setenv("BP_MAVEN_DAEMON_ENABLED", "true")
```

Comment:
Same here. We want this to be its own Context with a Before/After so that the env variables are properly cleaned up.